.github/workflows/pre_commit_suggestions.yaml

```
@@ -60,3 +60,16 @@ jobs:
           cat ./diff | \
           GITHUB_EVENT_PATH=./event \
             reviewdog -f=diff -f.diff.strip=1 -reporter=github-pr-review
+
+  delete-stale-suggestions:
+    if: |
+      github.event.workflow_run.event == 'pull_request' &&
+      github.actor != 'jonmeow'
```

```suggestion
      github.event.workflow_run.event == 'pull_request'
```

Don't think I need to be mentioned here

---

Done.